Document how to specify view permissions

not-raspberry opened the pull request:
### What does this PR do?
It adds documentation to explain how to specify view permissions.

Example:

`pg_stat_database` is a view in `pg_catalog`.

```
datadog_grants:
  postgres_privileges.present:
    - name: datadog
    - prepend: pg_catalog
    - object_name: pg_stat_database
    - object_type: table
    - privileges:
      - SELECT
```

### What issues does this PR fix or reference?
None.

### Previous Behavior
There was no relevant documentation.

### New Behavior
There is documentation.

### Tests written?
No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

salt-jenkins commented:
@not-raspberry, thanks for your PR! By analyzing the history of the files in this pull request, we identified @akissa, @eliasp and @gtmanfred to be potential reviewers.

rallytime commented:
Welcome @not-raspberry! Thank you for updating this documentation. It looks like there is a small lint error (extra whitespace) in your change. Would you mind fixing that up? Then we can get this merged in. You can see the error here: https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/12742/violations/file/salt/states/postgres_privileges.py/

not-raspberry committed:
Document how to specify view permissions

Example:

`pg_stat_database` is a view in `pg_catalog`.

```
datadog_grants:
  postgres_privileges.present:
    - name: datadog
    - prepend: pg_catalog
    - object_name: pg_stat_database
    - object_type: table
    - privileges:
      - SELECT
```
